app/src/main/java/org/schabi/newpipe/App.java

```diff
@@ -246,9 +246,16 @@ private void initNotificationChannels() {
                 .setDescription(getString(R.string.hash_channel_description))
                 .build();
 
+        final NotificationChannelCompat errorReportChannel = new NotificationChannelCompat
+                .Builder(getString(R.string.error_report_channel_id),
+                NotificationManagerCompat.IMPORTANCE_LOW)
```

Comment:
Shouldn't there be an indentation

Reply:
It also looks strange to me, but I just copied the formatting from the other notification channels above. Should I change them all?

Reply:
I don't know.
Do it if you want :)